imports/plugins/core/checkout/client/components/cancelOrderButton.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const CancelOrderButton = ({ cancelOrder, order }) => {
+	  return (
+    <div>
+      {
+        order.workflow.status !== "coreOrderWorkflow/canceled" ?
```

Comment:
As the false value of the ternary operator is an empty string, you can write this section as
```jsx
{order.workflow.status !== "coreOrderWorkflow/canceled" && (
  <div>
    <div className="cancel-order-button">
      ...
    </div>
  </div>
)}